apps/web/components/ReviewsAccordion/ReviewsAccordion.vue

```diff
@@ -86,8 +94,8 @@ const { t } = useI18n();
 const { isOpen: isReviewOpen, open: openReviewModal, close: closeReviewModal } = useDisclosure();
 const { isAuthorized } = useCustomer();
 const { isOpen: isAuthenticationOpen, open: openAuthentication, close: closeAuthentication } = useDisclosure();
-const reviewsOpen = ref(false);
-
+const reviewsOpen = ref(true);
```

Comment:
This is what @doproiescu-plenty is working on. I would revert this to not have conflicts.